packit/schema.py

```diff
@@ -156,6 +156,7 @@ class PackageConfigSchema(Schema):
     downstream_package_name = fields.String()
     upstream_project_url = fields.String(missing=None)
     upstream_package_name = fields.String()
+    archive_name = fields.String()
```

Comment:
since I can see some templating being done below, can you document here some expected values? or in package config, doesn't matter where